docs/release-notes/version-2.7.md

```diff
@@ -2,6 +2,7 @@
 
 ## Enhancements
 
+* [#568](https://github.com/netbox-community/netbox/issues/568) - Allow custom fields to be imported and exported using CSV
```

Comment:
This is fine as-is, but don't worry about adding changelog notes for future PRs. It's not worth the hasssle trying to avoid merge conflicts given all the recent activity. I'll make sure they get added.

Reply:
Thank you.